main.go

```diff
@@ -120,6 +122,10 @@ func main() {
 		"The TTL of an index in the cache. Valid time units are ns, us (or µs), ms, s, m, h.")
 	flag.StringVar(&helmCachePurgeInterval, "helm-cache-purge-interval", "1m",
 		"The interval at which the cache is purged. Valid time units are ns, us (or µs), ms, s, m, h.")
+	flag.DurationVar(&artifactRetentionTTL, "artifact-retention-ttl", 30*time.Second,
```

Comment:
I would set the default TTL to `60s` that matches the pull interval default set by bootstrap.